contributor-guide/modules/ROOT/pages/getting-involved.adoc

```diff
@@ -53,7 +53,7 @@ All libraries are reviewed by the community before becoming part of Boost. There
 
 If you have a deep interest in a particular library that for some reason no longer has an active owner or maintainer, or is being less well maintained than might be ideal (find this out by asking questions on the https://lists.boost.org/mailman/listinfo.cgi/boost[Boost developers' mailing list]), then consider requesting, again via the mailing list, to become a library maintainer. For larger libraries there can certainly be more than one maintainer. 
 
-Maintaining an existing library is similar in steps to <<3. Contribute to an Existing Library>>, in that you will have to create a fork and clone the library. However, more is expected. You will have to spend time educating yourself on the full functionality of the library, and be prepared to make changes to the code anywhere changes are needed. 
+Maintaining an existing library is similar in steps to <<Contribute to an Existing Library>>, in that you will have to create a fork and clone the library. However, more is expected. You will have to spend time educating yourself on the full functionality of the library, and be prepared to make changes to the code anywhere changes are needed. 
 
 For a larger library this education can turn out to be a significant, and perhaps onerous task. It can be quite difficult to reverse engineer in your mind the processes and purpose of code written by another developer, with all the expected personal style and idiosyncrasies that make their way into source code. However, that being said, it can be a valuable contribution to take an existing but deprecated library you are interested in and give it a new lease of life.
```

Comment:
> deprecated library

Libraries are not really deprecated. They might not be properly maintained (new features, ...) but not deprecated. They are all tested when the super-project is tested.

Reply:
OK, got it, reworded